kobo/apps/stripe/tests/test_stripe_utils.py

```
@@ -694,3 +699,30 @@ def test_check_exceeded_limit_cache_restriction(self):
             ) as patched:
                 check_exceeded_limit(self.someuser, UsageType.SUBMISSION)
                 patched.assert_called_once()
+
+    def test_update_or_remove_limit_counter(self):
+        mock_balances = {
+            UsageType.SUBMISSION: {'exceeded': True},
+        }
+        counter = baker.make(
```

Does it really make fast to use `baker.make()` rather than `ExceededLimitCounter.objects.create()`?

---

I can't say I know too much about the performance comparisons. I doubt we're going to be saving much time here either way. I tend to use model bakery by default when it's available, since it's nice not have to think about every field that might not be optional in a model.

---

I think the opposite because it can silently hide a problem.